agenta-cli/agenta/__init__.py

```diff
@@ -22,7 +23,7 @@
 from .sdk.utils.helper.openai_cost import calculate_token_usage
 from .sdk.client import Agenta
 from .sdk.tracing import callbacks
-
+from .sdk.config_manager import ConfigManager
```

Comment:
I'd definitely call this `Config`, similar to `Tracing`. But I guess it just a preference without arguments.

Reply:
hmm.. I would go here with the most explicit option. I'd say that `Tracing` is actually similar to `ConfigManager` (the concept)  while `Trace` would be similar to `Config` (the object)

Reply:
You're right.